src/styles/blocks/cardBuy.scss

```diff
@@ -0,0 +1,24 @@
+.card__buy {
+  box-sizing: border-box;
+  height: 40px;
+  width: 166px;
+  text-decoration: none;
+  text-transform: uppercase;
+  text-align: center;
+  background: $link-bg-color;
+  color: $link-text-color;
+  border-radius: 5px;
+  margin-top: 0;
+  font-size: 14px;
+  font-weight: 700;
+  line-height: 40px;
+  border: 1px solid $link-bg-color;
+
+  &:hover {
+    box-sizing: border-box;
```

Comment:
The `box-sizing` property is already declared in the `.card__buy` block (line 2), so it's unnecessary to repeat it in the `&:hover` state. Duplicated properties increase the file size and can lead to maintenance issues.